[Dominion] Fix profile tabs hrefs + map zoom

KhadijaMahanga opened the pull request:
## Description
This PR fixes profile tabs hrefs for article section in profile_detail page, and only displays profile tabs for existing article section for a profile. It also zooms in featured geography below country level.

Fixes # (issue)
- [x] Bug fix (non-breaking change which fixes an issue)


## Screenshots
![Screenshot from 2019-04-01 09-06-35](https://user-images.githubusercontent.com/7962097/55306607-81547b80-545d-11e9-9125-abacc0ba94d8.png)
![Screenshot from 2019-04-01 09-05-39](https://user-images.githubusercontent.com/7962097/55306612-83b6d580-545d-11e9-8e4e-565e6134c6fc.png)
![Screenshot from 2019-04-01 09-02-25](https://user-images.githubusercontent.com/7962097/55306617-887b8980-545d-11e9-81e1-7803df3a24bb.png)


## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

KhadijaMahanga committed:
Append profile tab for each article component

KhadijaMahanga committed:
Append profile tab for each article component

KhadijaMahanga committed:
remove statics profiletabs array

KhadijaMahanga committed:
styling of profile tabs

KhadijaMahanga committed:
zoom featured level

KhadijaMahanga committed:
remove redundant style props

karimkawambwa commented:
> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

@KhadijaMahanga

karimkawambwa commented on dominion/static/js/profile_map_mapit.js:
```diff
@@ -129,7 +129,8 @@ var ProfileMaps = function() {
         this.map.addLayer(layer);
-        this.map.setView(this.dominion_country.centre, this.dominion_country.zoom);
+        this.map.fitBounds(layer.getBounds());
+      //  this.map.setView(this.dominion_country.centre, this.dominion_country.zoom);
```
remove this commented out code if its not useful anymore

karimkawambwa reviewed (commented):
What sort of scroll behaviour are you going for ?
I don't see smooth scrolling and the tabs are hidden at the top when scrolling to profile section so you have to scroll all the way back.

DavidLemayian reviewed (changes requested):
@KhadijaMahanga Please add description and screenshots.

KhadijaMahanga committed:
removed commented code

karimkawambwa reviewed (approved):
From the description, works as intended 🚢 it.

kilemensi committed:
Filter instead of navigating

 + Hide every other tab,
 + Show selected tab, and
 + Make sure the tabs are visible on click.

KhadijaMahanga committed:
Merge branch 'master' into fix/feature-profiletabs

KhadijaMahanga committed:
Merge branch 'fix/feature-profiletabs' of github.com:CodeForAfrica/HURUmap-apps into fix/feature-profiletabs
